scripts/libs.js

```diff
@@ -15,6 +15,51 @@ export const globalConfig = {
   },
 };
 
+export const metaTags = {
+  componentsConfig: {
+    metaName: 'components-config',
+    fallbackContent: 'components-config.json',
+    // contentType: 'path without extension',
+  },
+  header: {
+    metaName: 'header',
+    fallbackContent: 'header.plain.html',
```

Comment:
I think we can skip the `.plain.html` here because we can/should statically append this when loading